R/get_plot_forecast_data.R

```diff
@@ -61,7 +61,7 @@ get_plot_forecast_data <- function(forecast_data,
       "cum death", "inc case",
       "inc death", "inc hosp"
     )
-    valid_truth_sources <- c("JHU", "USAFacts", "NYTimes", "HealthData")
```

Comment:
I'm not sure it'd ever be triggered, but in case someone tried to run this code later, we could leave USAFacts as a valid source but add a specific error message here saying something like "USAFacts data can no longer be downloaded. Please use another source.".

Reply:
That's a great point! Even if it's not clear anyone would be looking for it, I'd like to get in the habit of erring on the side of user-friendliness (and respecting the R mentality of avoiding breaking changes where possible).

I'll make this change.